src/GitHub.InlineReviews/ViewModels/CommentViewModel.cs

```diff
@@ -130,22 +131,31 @@ protected void AddErrorHandler<T>(ReactiveCommand<T> command)
 
         async Task DoDelete(object unused)
         {
-            try
-            {
-                ErrorMessage = null;
-                IsSubmitting = true;
+            var result = MessageBox.Show(
```

Comment:
We shouldn't be showing a message box from the view model because this will make message boxes pop up during unit testing, breaking things. This will need to be extracted out to a service.

Reply:
I'm also not sure if we should be using `VsShellUtilities.ShowMessageBox` instead of `MessageBox.Show` in VS? @AArnott any guidance?

Reply:
`VsShellUtilities.ShowMessageBox` just calls into [`IVsUIShell::ShowMessageBox`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.interop.ivsuishell.showmessagebox?view=visualstudiosdk-2017) I'm pretty sure. You're welcome to call that interface directly and mock it up in your unit tests, or to use the `VsShellUtilities` method if you're already mocking up the global service provider.

Reply:
Thanks @AArnott - that's definitely a good option. The question was actually regarding `MessageBox.Show` vs the VS-specific message-box methods, but given that we can pass in a `IVsUIShell` instance I'm not sure that's relevant any more ;)